### PR TITLE
Add pvandyk Indianapolis recommendations

### LINKED ISSUE #13: Add file for pvandyk's Hometown
- [x] Create a branch
- [x] Make commits
- [x] Create a Pull Request
- [x] Collaborate
- [x] Review
- [x] Merge

### PR DESCRIPTION
**Pull request by pvandyk**
closes #13

### CODE REVIEW

**Comment by apschwartz** [comment]
Hi peter.  Be sure to include some good steak places (for us carnivores).

**Comment by gmontano671** [comment]
:octocat:

File: pvandyk-Indianapolis.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Indianapolis
+
+### Places to Eat
```
Comment (gmontano671) [inline pvandyk-Indianapolis.md:3]: are any of these good steak places?

File: pvandyk-Indianapolis.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Big Lug
+- Bakersfield
+- City Market
+
+### Places to Visit
+- Canal
+- IMS
+- City Market
+
+### Places to Avoid
```
Comment (gmontano671) [inline pvandyk-Indianapolis.md]: any places to avoid?

Comment (gmontano671) [reply pvandyk-Indianapolis.md]: 👨‍🚀

**Review by gmontano671 — COMMENTED** [review]
general comment

**Review by gmontano671 — APPROVED** [review]

**Pull request merged**